dev/acf-export.json

```diff
@@ -145,6 +145,27 @@
                 "append": "",
                 "maxlength": ""
             },
+            {
+                "key": "field_5bcf50f2ba743",
+                "label": "Topic Sort Order",
+                "name": "topic_sort_order",
+                "type": "number",
+                "instructions": "Enter a number to sort the topics other than alphabetical",
```

Comment:
To help describe what exactly the sort value is for, I'd suggest re-wording this field description to something like: "Enter an order value by which this topic should be sorted when displayed in a topic list.  Topics with lower sort order values will be placed at the top of the list.  If a value isn't provided, this topic will be sorted alphabetically."